pkg/pillar/agentlog/agentlog.go

```diff
@@ -157,11 +174,11 @@ func printStack() {
 // Note: can not use log here since we are called from a log hook!
 func RebootReason(reason string, normal bool) {
 	log.Infof("RebootReason(%s)", reason)
-	filename := fmt.Sprintf("%s/%s", getCurrentIMGdir(), reasonFile)
+	filename := fmt.Sprintf("%s/%s", types.PersistDir, reasonFile)
```

Comment:
This means we should clean up zedagent (or is it nodeagent) where it gets the reason/stack/time.  But we need to keep the current code there for a while since the other partition might be an older release.

We could replace the GetCurrentReboot* calls with GetCommonReboot* calls and delete the GetCurrentReboot*, while keeping the GetOtherReboot* calls.

Reply:
will do

Reply:
done